docker-compose.yml

```diff
@@ -38,14 +38,14 @@ services:
     volumes:
       - redis:/data
 
-  postgres:
+  db:
     env_file: docker/.env
     image: postgres:10
     restart: unless-stopped
     ports:
       - "127.0.0.1:5432:5432"
     volumes:
-      - postgres:/var/lib/postgresql/data
+      - db_home:/var/lib/postgresql/data
```

Comment:
should this be abstracted too? `/var/lib/sql_db/data` to avoid confusion

Reply:
You can't without a duplicate `.env` file, or running docker-compose with an env switch